html/portfolio1.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+
+<!--LINKS-->
+    <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Heebo" rel="stylesheet">
+    <link href="../css/portfolio.css" rel="stylesheet" type="text/css">
+
+    <title>Alice McNeil</title>
+
+</head>
+
+<body>
+<div>
+<!--LOGO-->
+    <img src="../icons/alice-logo.png" class="logo" alt="alice logo">
+
+    <a class="triangle" name="home"></a>
+
+
+
+<!--NAVBAR-->
+
+    <nav class="navbar">
+        <a href="#home" class="home">Home</a>
+        <a href="#projects" class="work">My Projects</a>
+        <a href="#contact" class="contact-button">Contact</a>
+    </nav>
+
+
+<!--MAIN HEAD-->
+    <header class="width main-head">
+        <h2>Alice McNeil</h2>
+        <h1>Portfolio</h1>
+    </header>
+
+
+    <div class="top-section width">
+<!--IMAGE OF ME-->
+        <div class="profile-square"></div>
+        <img src="../images/me.png" class="me-pic" alt="image of alice">
+
+        <!--ABOUT-->
+        <div class="width about-me">
+            <h3 id="about">About me</h3>
+            <p>Hello, I am Alice McNeil; welcome to my Portfolio Page. Currently I am doing a software developer course at Mayden Academy,
+                I hope to learn the skills I will need to become a front-end software developer. I love graphic design, music and code!
+                Please take a look around at my portfolio.</p>
+        </div>
+    </div>
+
+<!--PORTFOLIO-->
+
+    <a name="projects" class="project-link"></a>
+
+    <div class="width my-work">
+
+        <h3>Here's some of my projects</h3>
+    </div>
+
+ <!--IMAGES-->
+        <div class="image-container width">
+            <img src="../images/pilotscreenshot.png" alt="pilotshop webpage" class="image1">
+            <img src="../images/pilotscreenshot.png" alt="pilotshop webpage" class="image2">
+            <img src="../images/pilotscreenshot.png" alt="pilotshop webpage" class="image3">
+        </div>
+
+
+
+    <div class="color-block">
+        <div class="dark-left"></div>
+        <div class="clear"></div>
+        <div class="light-right"></div>
+        <div class="white-low"></div>
+    </div>
+
+    
+
+    <!--CONTACT-->
+
+    <a name="contact" class="contact-link"></a>
+
+
+    <div class="width contact-section">
+        <h3 class="width">Contact me</h3>
+        <p class="get-in-touch">Well you've made it this far, you may as well get in touch with me.</p>
+        <p>EMAIL: alicemcneil@live.co.uk</p>
+        <p>Phone: 07902426939</p>
```

Comment:
I wouldn't put my phone number on a website, recruiters can be pretty persistent about trying to get you to work for them and it could also end up on a spam call list very quickly